warehouse/models/staging/gtfs_guidelines/stg_gtfs_guidelines__fact_daily_transitland_url_check.sql

```diff
@@ -0,0 +1,23 @@
+{{ config(materialized='table') }}
+
+WITH gtfs_fact_daily_transitland_url_check AS (
+    SELECT
+
+        dt,
+
+        itp_id,
```

Comment:
sorry for not thinking of this earlier, but I would rename these to `calitp_itp_id` and `calitp_url_number` just for consistency with the GTFS pipeline